templates/event-articles/gear-5-screening.njk

```diff
@@ -11,7 +11,7 @@
 	</div>
 	
 	<div class="image-container" style="width: fit-content;">
-		<img src="/assets/events/gear-5/1.png" alt="Gear 5 Poster" style="object-fit: contain;width: fit-content;">
+		<img src="/assets/events/gear-5/1.webp" alt="Gear 5 Poster" style="object-fit: contain;width: fit-content;">
```

Comment:
This commit with hash:
9973b326561ffd702ad5490caa6540e6a0868d05

Its not commenting on a the commit so a workaround.